docs/docs/contribute/.pages

```diff
@@ -0,0 +1,4 @@
+nav:
+  - index.md
+  - General
```

Comment:
No such page or folder.

A `.pages` file is only used where there is a need to customize the pages order. It's likely unnecessary here.

See https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin

Reply:
Ok, removed this file. Can add later if needed.